src/js/vendor/search-ui.js

```diff
@@ -194,13 +192,22 @@ window.antoraLunr = (function (lunr) {
     }
   }
 
-  function init (data) {
-    var index = Object.assign({index: lunr.Index.load(data.index), store: data.store})
+  function init (lunr, data) {
```

Comment:
For more readability 

```
function init(lunr, data) {
  const index = { index: lunr.Index.load(data.index), store: data.store };
  const isDebugEnabled = 'URLSearchParams' in window && new URLSearchParams(window.location.search).has('lunr-debug');

  const debouncedSearch = debounce(() => {
    try {
      searchIndex(index.index, index.store, searchInput.value);
    } catch (err) {
      if (isDebugEnabled) console.debug(`Invalid search query: ${searchInput.value} (${err.message})`);
    }
  }, 100);

  searchInput.addEventListener('keydown', debouncedSearch);
  searchInput.addEventListener('keydown', (event) => event.key === 'Escape' && (searchInput.value = ''));
  searchInput.addEventListener('blur', () => { while (searchResult.firstChild) searchResult.removeChild(searchResult.firstChild); });
}
```

Reply:
Unfortunately does not work the currently used version of GulpUglify.